### ISSUE TITLE
Error building: missing react-resize-detector

### ISSUE THREAD

**Issue by oyeanuj** [heydovetail/react-masonry-responsive#7]
Hi folks! Thank you for open-sourcing this library - looks great!

I run into the following error during the build:

```
ERROR in ./node_modules/react-masonry-responsive/lib.es2015/index.js
Module not found: Error: Can't resolve 'react-resize-detector' in '/project-okay/node_modules/react-masonry-responsive/lib.es2015'
 @ ./node_modules/react-masonry-responsive/lib.es2015/index.js 2:0-56 23:74-93
```
My setup is a classic webpack setup.

What would you suggest as the best fix?

**Comment by tylermurry** [comment]
I also ran into this. Wouldn't the best option be to include it in your dependencies?

**Comment by humphreybc** [comment]
Fixed and released in 2.1.2. Thanks @oyeanuj and @tylermurry!
